doc/modules/classes.rst

```diff
@@ -858,6 +858,7 @@ details.
    metrics.homogeneity_completeness_v_measure
    metrics.homogeneity_score
    metrics.mutual_info_score
+   metrics.cluster.contingency_matrix
```

Comment:
Please respect alphabetical order

Reply:
Okay, since we don't follow such rule in imports at the top of python files that's why I missed this.

Is everything else okay?